nixos/packages.tt

```diff
@@ -1,18 +1,18 @@
-[% WRAPPER layout.tt title='Search NixOS packages' menu='nixos' %]
```

Comment:
This diff is mostly updating single quotes to be more internet-standard double quotes, plus  `<span class="muted">$</span> my command` to `<span class="command">my command</span>` or two.